### PR TITLE
fix: avoid react-refresh error in test env

### PR DESCRIPTION
**Pull request by sanyuan0704**
# PR Details

Solve the following error in test environment:

img_v2_d3bf58e2-079b-4090-ad11-87829552fa8g

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by chenjiahan** [comment]
需要加个 changeset 哈

File: packages/cli/webpack/src/config/features/react-refresh.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function applyReactRefreshBabelPlugin({ chain }: ChainUtils) {
       ...options,
       plugins: [
         ...(options.plugins || []),
-        require.resolve('react-refresh/babel'),
+        [require.resolve('react-refresh/babel'), { skipEnvCheck: true }],
```
Comment (chenjiahan) [inline packages/cli/webpack/src/config/features/react-refresh.ts:24]: 是不是改错地方啦，这里改的是 `cli/webpack` 这个包，但是 changeset 写的是 @modern-js/webpack-builder

File: packages/builder/webpack-builder/package.json [hunk 2]
```diff
@@ -15,7 +14,7 @@
   "engines": {
     "node": ">=14.0.0"
   },
-  "version": "1.1.2",
+  "version": "0.0.0-fix-test-env.0",
```
Comment (chenjiahan) [inline packages/builder/webpack-builder/package.json:17]: 这个应该不用提交哈

Comment (sanyuan0704) [reply packages/builder/webpack-builder/package.json:17]: 给奎哥的测试版提上来了，我删一下

File: packages/cli/webpack/src/config/features/react-refresh.ts (hunk 1 above)
Comment (sanyuan0704) [reply packages/cli/webpack/src/config/features/react-refresh.ts:24]: 发现了

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**